python/lsst/summit/extras/logUtils.py

```diff
@@ -64,13 +77,23 @@ class LogBrowser:
         "with gufunc signature (n?,k),(k,m?)->(n?,m?)",
     ]
 
-    def __init__(self, butler, taskName, collection):
+    def __init__(self, butler, taskName, collection, whereStr="", bindMap=None):
```

Comment:
Personally, I'd call `bindMap` just `bind`, but then I'd probably also call `whereStr` just `where` tbh. I think these should be familiar terms for people, and also results in the common `where=where, bind=bind` pattern.